<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashierWindow</class>
 <widget class="QMainWindow" name="CashierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cashier Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="memberGroup">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Member</string>
    </property>
    <widget class="QRadioButton" name="memberNo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>NO</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="memberYes">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>YES</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addItemButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeItemButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipsLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="orderTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Goods Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit-Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tlwg Typo</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerMemberButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Register New Member</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cashierName">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(52, 101, 164);
}</string>
    </property>
    <property name="text">
     <string>cashier</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipsLabel1_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Receive:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="receiveInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipsLabel1_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change:</string>
    </property>
   </widget>
   <widget class="QLabel" name="changeLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Pending transaction</string>
    </property>
    <widget class="QPushButton" name="clearButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stageButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stage</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>memberGroup</zorder>
   <zorder>addItemButton</zorder>
   <zorder>amountLabel</zorder>
   <zorder>removeItemButton</zorder>
   <zorder>tipsLabel1</zorder>
   <zorder>orderTable</zorder>
   <zorder>finishButton</zorder>
   <zorder>registerMemberButton</zorder>
   <zorder>label</zorder>
   <zorder>cashierName</zorder>
   <zorder>tipsLabel1_2</zorder>
   <zorder>receiveInput</zorder>
   <zorder>tipsLabel1_3</zorder>
   <zorder>changeLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
